platforms/src/index.ts

```diff
@@ -9,3 +9,5 @@ export {
 } from "./Twitter/providers/TwitterFollowerProvider";
 export { TwitterTweetGT10Provider } from "./Twitter/providers/TwitterTweetsProvider";
 export { TwitterPlatformDetails, TwitterProviderConfig } from "./Twitter/Providers-config";
+
+export * as Ens from "./Ens";
```

Comment:
This export method for ens does make it hard to find the modules in the IAM. This is also leading errors in the CI